library/VTKExtensions/Rendering/vtkF3DRendererWithColoring.cxx

```diff
@@ -439,7 +436,7 @@ void vtkF3DRendererWithColoring::SetColoring(
       // Not coloring
       this->ArrayIndexForColoring = -1;
     }
-    else if (arrayName != F3D_RESERVED_STRING && nIndexes == 0)
+    else if (nIndexes == 0)
```

Comment:
this doesnt look right

Reply:
clang tidy reports that `arrayName != F3D_RESERVED_STRING` is always true.